src/cloudflare/internal/vectorize-api.ts

```diff
@@ -1,7 +1,14 @@
 // Copyright (c) 2023 Cloudflare, Inc.
 // Licensed under the Apache 2.0 license found in the LICENSE file or at:
 //     https://opensource.org/licenses/Apache-2.0
-import { default as flags } from 'workerd:compatibility-flags';
+declare global {
+  const Cloudflare: {
+    readonly compatibilityFlags: Record<string, boolean>;
```

Comment:
Can we explicitly define these values that we use?

Reply:
I'd prefer that we find a way of auto-generating the list from the source capnp file. Let's explore that first rather than adding another thing to manually update. I'm not in a hurry to land this.